src/views/TextAnnotator.vue

```diff
@@ -204,6 +230,7 @@ export default {
   methods: {
     back() {
       this.annotatedText = '';
```

Comment:
Is this still used? If not please remove

Reply:
The "annotatedText" variable contains all the data returned from SciGraph and is the "source of truth" so to speak. There are some issues with SciGraph not returning appropriate categories or no category at all (see #415 for example), so I created two variables: one to store the output from the API ("annotatedText") and one to store a modified version that strips out terms with blank categories and/or assigns a usable category for display in the UI ("annotTextCleaned"). The complete list of unmodified annotations from SciGraph is what is exported, so yes, "annotatedText" is still used.